test/Microsoft.ML.Tests/Scenarios/ClusteringTests.cs

```diff
@@ -1,128 +0,0 @@
-﻿using System;
-using System.Collections.Generic;
-using Microsoft.ML.Data;
-using Microsoft.ML.Legacy.Transforms;
-using Xunit;
-
-namespace Microsoft.ML.Scenarios
-{
```

Comment:
I am not positive those tests have an equivalent through the other API. 

I know the Scenarios/PipelineApi folder ones do.